src/CodeGen_LLVM.cpp

```diff
@@ -1951,7 +2032,7 @@ void CodeGen_LLVM::visit(const Load *op) {
 
         llvm::Type *load_type = llvm_type_of(op->type.element_of());
         if (ramp && stride && stride->value == 1) {
-            value = codegen_dense_vector_load(op);
+            value = codegen_dense_vector_load(op, nullptr);
```

Comment:
(1) nullptr is the default for `vpred` so this isn't necessary
(2) if it's desirable to be explicit anyway, please comment a la `/*vpred*/ nullptr`